src/typescript/azure-openapi-validator/rules/DescriptionAvoidsGerunds.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+
+export const DescriptionAvoidsGerunds: string = "DescriptionAvoidsGerunds";
```

Comment:
have you run this rule on the specs in the repo? how many warnings are getting flagged?
We don't have a way currently to filter per category so, I worry about the noise brought by these, if we don't introduce something like that.

Reply:
There hasn't been a test run on the full swagger yet. While all of these rules will undoubtedly flag a bunch of stuff, they have been required for that exact reason. Running on live data may be required for gerund detection, so that I can whitelist any other non-gerund `-ing` nouns.